Boundary documentation: added new metrics doc (#1977)
hashicorp/boundary: website/content/docs/operations/index.mdx

---
layout: docs
page_title: Operating Boundary
description: |-
  Operational tasks in Boundary.
---

> # Maintaining and Operating a Running Boundary

Author: Yes,I agree too with your suggestion :)

> Having visibility into various components of Boundary plays an important role in ensuring the health of its Controllers and Workers. While we intend to offer additional monitoring components in subsequent Boundary releases, Boundary can now add metrics to its Controllers and Workers and be collected by Prometheus. Collecting metrics helps you understand the current state of your Boundary deployment, thereby, enabling the diagnosis of deployed Controllers and Workers to help detect anomalies and mitigate. This section will cover the metrics available as well as the naming and labeling standards used for the metrics collected.

Reviewer: Boundary exports the metrics, it is up to the boundary operator to collect them and do something with them.

```suggestion
Having visibility into various components of Boundary plays an important role in ensuring the health of its Controllers and Workers. While we intend to offer additional monitoring components in subsequent Boundary releases, Boundary can now export metrics for its Controllers and Workers which can be collected by Prometheus.  These metrics help you understand the current state of your Boundary deployment, thereby, enabling the diagnosis of deployed Controllers and Workers to help detect anomalies and mitigate. This section will cover the metrics available as well as the naming and labeling standards used for the metrics exported.
```
